blocks/newsletter/includes/class-renderer.php

```diff
@@ -35,11 +35,11 @@ public function __construct( array $block_attributes ) {
 	 * @return string
 	 */
 	public function get_content() : string {
-		$form        = $this->get_form();
-		$legal_text  = $this->get_legal_text();
-		$classes     = $this->get_classes(
+		$form         = $this->get_form();
+		$legal_text   = $this->get_legal_text();
+		$wrapper_attr = get_block_wrapper_attributes(
 			array(
-				'wp-block-cata-newsletter',
+				'class' => 'wp-block-cata-newsletter',
```

Comment:
Does it automatically produce the same class name that we're assigning here?

Reply:
Yep it does, removed